ui/app/pages/confirm-transaction/confirm-transaction.component.js

```diff
@@ -45,6 +45,7 @@ export default class ConfirmTransaction extends Component {
     isTokenMethodAction: PropTypes.bool,
     fullScreenVsPopupTestGroup: PropTypes.string,
     trackABTest: PropTypes.bool,
+    conversionRate: PropTypes.number,
```

Comment:
Sorry for the post-merge nitpick, but this prop appears to be unused (both here and in the container component)